software/webapp/src/main/java/brooklyn/entity/webapp/nodejs/NodeJsWebAppSshDriver.java

```diff
@@ -103,6 +103,7 @@ public void install() {
                         BashCommands.installPackage("software-properties-common python-software-properties python g++ make"),
                         BashCommands.sudo("add-apt-repository ppa:chris-lea/node.js"))))
                 .add(BashCommands.installPackage(MutableMap.of("yum", "git nodejs npm", "apt", "git-core nodejs"), null))
+                .add("npm install n") // Create ~/.npm as the current user instead of root
```

Comment:
I thought that global install was not recommended, should use normal when possible? Not a NodeJS expert though so might as well be wrong.